schemas/oic.r.coapcloudconf-update-schema.json

```diff
@@ -1,38 +1,33 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
-  "id": "https://www.openconnectivity.org/ocf-apis/core/schemas/oic.r.coapcloudconf-update-schema.json#",
+  "id": "https://www.openconnectivity.org/ocf-apis/core-extensions/schemas/oic.r.coapcloudconf-update-schema.json#",
   "definitions": {
     "oic.r.coapcloudconf": {
       "type": "object",
       "properties": {
         "apn": {
           "type": "string",
-          "description": "Authentication provider's name issuing the auth code",
+          "description": "The Authorisation Provider through which an Access Token was obtained.",
           "pattern": "^.*$"
         },
         "cis": {
           "type": "string",
-          "description": "Indicates URL for CoAP native cloud interface server to be registered",
+          "description": "URL of OCF Cloud",
           "format": "uri"
         },
         "at": {
           "type": "string",
-          "description": "Indicates access token which is given in return of auth code issued by an appropriate account server",
-          "pattern": "^.*$"
-        },
-          "sid": {
-          "type": "string",
-          "description": "The Server identity of the OCF Cloud used for verifying the OCF Cloud certificate.",
+          "description": "Access Token which is returned by an Authorisation Provider or OCF Cloud.",
           "pattern": "^.*$"
         }
       },
-      "required":["cis"]
+      "required":["cis", "at"]
     }
   },
   "type": "object",
   "allOf": [
-    { "$ref": "oic.core-schema.json#/definitions/oic.core"},
+    { "$ref": "../../core/schemas/oic.core-schema.json#/definitions/oic.core"},
```

Comment:
We  can't have "..." paths in our schemas.  I heard Wouter mention this on the last DMWG call.

Reply:
Same as above.